pkg/providers/amifamily/bootstrap/bootstrap.go

```diff
@@ -15,7 +15,17 @@ limitations under the License.
 package bootstrap
```

Comment:
nit: label this file `args.go` since i has to do with passing args to the bootstrap scripts

Reply:
ok.